cmd/notary/tuf.go

```diff
@@ -268,7 +275,36 @@ func (t *tufCommander) tufInit(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
-	rootKeyList := nRepo.CryptoService.ListKeys(data.CanonicalRootRole)
+	var rootKeyList []string
+
+	if t.rootKey != "" {
```

Comment:
Definitely, and I have a note on my desk to make this exact change and just forgot. I can make a new PR if appropriate